### PR TITLE
ENDOC-385 tips-and-tricks sheet needs to be reworked/augmented

### PR DESCRIPTION
**Pull request by Lyd1aCla1r3**
nothing added yet, just revisiting current content

### CODE REVIEW

File: vuepress/docs/v6.3.2/docs/reference/local-tips-and-tricks.md [hunk 1]
```diff
@@ -69,6 +69,31 @@ The base domain configured via the ENTANDO_DEFAULT_ROUTING_SUFFIX (e.g. in your
 ### `The IP address changed after the initial install`
 - Restarting a Windows computer can cause this (see [Windows Hyper-V IP Changes](#hyper-v-ip-changes) below), and the workaround noted above (e.g. update your /etc/hosts file) also applies. Simply update the IP address in the first column to use the current IP of your virtual machine. 
 
+## Customizing NGINX
+
+In some situations the default NGINX ingress configuration doesn't work well for Entando. For instance, JWT tokens can be too large, proxy-buffer-size can be too small, etc. A 502 Bad Gateway error can indicate this config needs to be modified.
+
+To configure the NGINX controller globally (for the entire cluster) we just need to edit the default NGINX's configmap, which is ingress-nginx-controller in the ingress-nginx namespace. Add the following inside the data parameter:
```
[the remark this answers is not in the document]
Comment (Lyd1aCla1r3) [reply vuepress/docs/v6.3.2/docs/reference/local-tips-and-tricks.md:92]: just confirming: do we capitalize configmap if it's not an Entando ConfigMap? and ConfigMap is one word?

File: vuepress/docs/v6.3.2/docs/reference/local-tips-and-tricks.md [hunk 2]
```diff
@@ -69,6 +69,31 @@ The base domain configured via the ENTANDO_DEFAULT_ROUTING_SUFFIX (e.g. in your
 ### `The IP address changed after the initial install`
 - Restarting a Windows computer can cause this (see [Windows Hyper-V IP Changes](#hyper-v-ip-changes) below), and the workaround noted above (e.g. update your /etc/hosts file) also applies. Simply update the IP address in the first column to use the current IP of your virtual machine. 
 
+## Customizing NGINX
+
+In some situations the default NGINX ingress configuration doesn't work well for Entando. For instance, JWT tokens can be too large, proxy-buffer-size can be too small, etc. A 502 Bad Gateway error can indicate this config needs to be modified.
```
[the remark this answers is not in the document]
Comment (Lyd1aCla1r3) [reply vuepress/docs/v6.3.2/docs/reference/local-tips-and-tricks.md:90]: i'm not consistent with this... is there a rule about adding words like "that" where they can go but are often skipped?

File: vuepress/docs/v6.3.2/docs/reference/local-tips-and-tricks.md [hunk 3]
```diff
@@ -10,18 +10,39 @@ practices over on the [Entando forum](https://forum.entando.org).
...
-1. **What do I do if Entando doesn't fully initialize?** The most common cause of this is a networking problem. See the [Network issues](#network-issues) section below for details. If all else fails, reach out to the Entando team on Slack or in the Forums. 
+**Q: How can I install a new copy of Entando into an existing VM?** 
+
+**A:** By default, the quickstart installation deploys Kubernetes resources into a dedicated namespace called `entando`. If you want to remove all of the resources in `entando` you can simply delete the namespace with `sudo kubectl delete namespace entando`. You can then re-create the namespace and re-install the resources by applying the Helm template for your environment. Alternatively, you can achieve this with `ent quickstart --vm-reuse=true`, but you'll need to set other `ent quickstart` options, so check the `ent` help.
+
+**Q: How can I shell into a running pod or view its logs?** 
+
+**A:** You can use the standard Kubernetes commands, e.g. `sudo kubectl exec -it <POD-NAME> -c <CONTAINER-NAME -- bash` or `sudo kubectl logs <POD-NAME> <CONTAINER-NAME>`
```
[the remark this answers is not in the document]
Comment (Lyd1aCla1r3) [reply vuepress/docs/v6.3.2/docs/reference/local-tips-and-tricks.md:57]: perhaps :P good catch!

**Review by nshaw — APPROVED** [review]
Looks good to me

File: vuepress/docs/v6.3.2/docs/reference/local-tips-and-tricks.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (jyunmitch) [reply vuepress/docs/v6.3.2/docs/reference/local-tips-and-tricks.md]: this is kind hazy to me too. I added it by sound. From what i can find, 'this config needs...' works as an object of the sentence and thus needs a 'that'. 

here's the best explanation i can find, https://web.ku.edu/~edit/that.html

**Review by jyunmitch — APPROVED** [review]

**Pull request merged**